lms/djangoapps/certificates/queue.py

```diff
@@ -585,3 +598,19 @@ def _send_to_xqueue(self, contents, key, task_identifier=None, callback_url_path
             exc = XQueueAddToQueueError(error, msg)
             LOGGER.critical(unicode(exc))
             raise exc
+
+    def _log_pdf_cert_generation_discontinued_warning(self, student_id, course_id, cert_status, download_url):
+        """Logs PDF certificate generation discontinued warning."""
+        LOGGER.warning(
+            (
+                u"PDF certificate generation discontinued, canceling "
+                u"PDF certificate generation for student %s "
```

Comment:
using format with variables would improve the readability.

Reply:
@adeelehsan Python does not allow 'str.format()' in logging.